restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/Starting Restcomm-Connect.adoc

```diff
@@ -1,14 +1,14 @@
 [[start-restcomm-connect]]
-*Starting Restcomm-Connect*
+= Starting Restcomm-Connect
 
-*Requirements*
+== Requirements
 
 *  Install the GNU screen (it is optional but recommended)
 
 * Install IPCalc
 * You must have *JDK - 7* installed on your computer
 
-*Step 1 – Download and unzip Restcomm binary*
+== Download and unzip Restcomm binary
 
 _If you you have already installed Restcomm you can skip the Step 1 and move to Step 2._
 
```

Comment:
You mention step 1 and Step 2 but there is no Steps anymore, feel free to make use of [asciidoc cross references](http://asciidoctor.org/docs/user-manual/#xref)